src/presence.rs

```diff
@@ -35,6 +35,8 @@ pub fn run_presence_updates(sqvm: NonNull<HSquirrelVM>) {
     };
 }
 
+// "Localize_001" is a thing
+
```

Comment:
Leftover comment or added annotation on purpose?

Reply:
for future stuff